proposed/cli-acquisition-tool.md

```diff
@@ -0,0 +1,149 @@
+
+### Acquisition inside VS Code
+
+Hector installs the C# DevKit extension on an otherwise-clean Windows machine. The DevKit extension recognizes the lack of a .NET SDK install and prompts Hector for permission to install the latest one. Hector accepts, and the DevKit extension downloads and runs the latest version of the dnup-init bootstrapper, instructing it to install the latest .NET 10 SDK to `%LOCALAPPDATA%\dotnet`, as well as updating Hector's user-level PATH. DevKit then continues initialization and Hector is able to begin development.
+
+### Acquisition with an LLM
+
+Ricardo wants to check what .NET versions have security vulnerabilities and update them. He asks his local GitHub Copilot which installed .NET versions are out of date. The agent talks to an acquisition MCP server that lets him know he has two older .NET versions that have known vulnerabilities and asks him if he wants to update. Then it calls `dnup update` to update to the latest version for each.
+
+### Onboarding to a repository using .NET
+
+Mei wants to contribute to a .NET library that she found on GitHub. She clones the repository, but the repo uses a global.json to enforce a certain tooling version for all contributors be installed to a repo-local directory. Since Mei doesn't have that version, she runs `dnup install` (or her editor does so on her behalf) and that version is downloaded and installed into her local repository copy. After this install is done, the next time she launches her editor it detects the install and she begins work on the feature she's selected.
+
+### Updating all installed toolchains
+
+Gavin has a few different SDKs managed via `dnup install` on their machine in the following configuration:
+
+- 8.0.404 - pinned by a `global.json` in `~/repos/mylib`
+- 9.0.303 - installed by tracking the 'latest' channel from `dnup install`
+- 10.0.100-preview.5 - installed tracking the 'preview' channel from `dnup install`
+
+They want to make sure their machine has all of the latest releases, so they run `dnup update`, which discovers:
+
+- there is a 8.0.405 version available
+- there is a 9.0.304 version available
+- there is a 10.0.100-preview.6 version available
+
+Since 8.0.404 is pinned by a specific repo, that version is not updated/removed, but 9.0.303 and 10.0.100-preview.5 are updated to 9.0.304 and 10.0.100-preview.6 respectively.
+
+## Requirements
+
+### Goals
+
+- **Unified SDK Management**: Provide a single tool for acquiring, updating, and managing .NET SDKs across all supported platforms
+- **User-local Installation**: Focus on user-writable locations that can be managed without elevation after initial setup
```

Comment:
Should we add that in the long term this may replace the install scripts/uninstall tool on mac but that we don't see it replacing the install scripts at least in the near-term? Similar with the vscode extension logic.